index.html

```diff
@@ -429,6 +429,14 @@
 					}
 				}),
 				document.getElementById("fw_form"));
+			        if (Math.round(100 * (data.system.sketchSpaceUsedBytes / data.system.sketchSpaceTotalBytes)) >= 75)
+				{
+					document.getElementById("root_size").style.borderColor = "red";
+				}
+				else
+				{
+					document.getElementById("root_size").style.borderColor = "";
```

Comment:
Much cleaner! Do we need the `else` statement, or does it not get cleared on a new device?

Reply:
`else` does need to be there to clear the style after a device that has set style red. the only time it would fully reset would be if you did `F5`